test/built-ins/Temporal/Duration/prototype/add/argument-mixed-sign.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.duration.prototype.add
+description: Positive and negative values in the temporalDurationLike argument are not acceptable
+features: [Temporal]
+---*/
+
+const instance = new Temporal.Duration(1, 2, 3, 4, 5, 6, 7, 987, 654, 321);
```

Comment:
The `relativeTo` option is required if the duration has calendar units in it — we should either zero out the years, months, and weeks, or add a `relativeTo` option. Otherwise we can't tell if the RangeError is occurring because of the missing `relativeTo` or the mixed signs.